test-harness/specs/validate-body-params/json-inference-invalid-request.pc.txt

```diff
@@ -77,9 +77,9 @@ Prism can reject the request which is not compatible with any of example bodies
 ====server====
 mock -p 4010 ${document}
 ====command====
-curl -i -X POST http://localhost:4010/path -H "Content-Type: application/json" --data '{"to":"123","type":"image","recipient_type":"individual"}'
+curl -i -X POST http://localhost:4010/path -H "Content-Type: application/json" --data '{"to":123,"type":"image","recipient_type":"individual"}'
 ====expect====
 HTTP/1.1 422 Unprocessable Entity
 content-type: application/problem+json
 
-{"type":"https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY","title":"Invalid request","status":422,"detail":"Your request is not valid and no HTTP validation response was found in the spec, so Prism is generating this error for you.","validation":[{"location":["body"],"severity":"Error","code":"required","message":"must have required property 'image'"},{"location":["body"],"severity":"Error","code":"required","message":"must have required property 'text'"},{"location":["body"],"severity":"Error","code":"anyOf","message":"must match a schema in anyOf"}]}
+{"type":"https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY","title":"Invalid request","status":422,"detail":"Your request is not valid and no HTTP validation response was found in the spec, so Prism is generating this error for you.","validation":[{"location":["body","to"],"severity":"Error","code":"type","message":"must be string"},{"location":["body","to"],"severity":"Error","code":"type","message":"must be string"},{"location":["body"],"severity":"Error","code":"anyOf","message":"must match a schema in anyOf"}]}
```

Comment:
our fork of json-schema-generator is more forgiving with generating json schema for an example. Our non-forked version made all properties required if they were in the example, our fork does not. So these tests were updated to reflect that.